website/docusaurus.config.js

```diff
@@ -45,12 +45,12 @@ const config = {
     /** @type {import('@docusaurus/preset-classic').ThemeConfig} */
     ({
       navbar: {
-        title: 'OneTable',
+        title: 'XTable',
         logo: {
-          alt: 'OneTable Logo',
+          alt: 'XTable Logo',
           href: 'pathname:///',
           target: '_self',
-          src: 'images/OneTableLogo.png',
+          src: 'images/xtable-icon.png',
```

Comment:
We should get icons updated someplace with asf I think.

Reply:
https://www.apache.org/logos/about.html this is what I was talking about. we can do it separately from this PR.

Reply:
ok, I will research the policy and add if there is a special asf place to store logos